app/assets/stylesheets/_buttons.scss

```diff
@@ -11,7 +11,7 @@
   -webkit-font-smoothing: antialiased;
   font-weight: 600;
   line-height: 1;
-  padding: 0.75em 1em;
+  padding: ($small-spacing / 1.2) ($base-spacing / 1.2);
```

Comment:
If these kinds of arbitrary measurements are to be made, I prefer them in straight `ems`, but even that is personal preference